File: samples/java-alt-resource-listing/src/main/java/com/wordnik/swagger/sample/resource/ApiListingResource.java
package com.wordnik.swagger.sample.resource;

import com.wordnik.swagger.annotations.*;
import com.wordnik.swagger.jaxrs.*;

import javax.ws.rs.*;

@Path("/resources")
@Api("/resources")
@Produces({"application/json","application/xml"})
public class ApiListingResource extends JavaApiListing {}

File: modules/swagger-jaxrs/src/main/scala/com/wordnik/swagger/jaxrs/listing/ApiListingResource.scala
package com.wordnik.swagger.jaxrs.listing

import com.wordnik.swagger.annotations._
import com.wordnik.swagger.jaxrs._

import javax.ws.rs._
import javax.ws.rs.core.MediaType

@Path("/api-docs")
@Api("/api-docs")
@Produces(Array("application/json; charset=utf-8", "application/json"))
class ApiListingResourceJSON extends ApiListingResource
